app/components/results-list/template.hbs

```diff
@@ -30,7 +30,7 @@
             </div>
         </div>
     {{else}}
-        <h3 style="text-align: center;">
+        <h3 class="results-text">
```

Comment:
You can actually do both the css rules in this class by helper classes (to avoid adding file) 
`<h3 class="text-center p-b-md">`

md is 15px
lg is 25px 
It might need to be strictly 20, in which case we can use the class.